dnf-plugins-core.spec

```diff
@@ -1,6 +1,6 @@
 %{?!dnf_lowest_compatible: %global dnf_lowest_compatible 1.1.4}
 %{?!dnf_not_compatible: %global dnf_not_compatible 2.0}
-%global hawkey_version 0.6.1
+%global hawkey_version 0.7.0
```

Comment:
This version does not exist in Fedora, and the latest possible hawkey version that can be introduced to Fedora is 0.6.3, which is also currently not in Fedora.